lib/management/authz.ts

```diff
@@ -223,6 +224,21 @@ const WithAuthz = (sdk: CoreSdk, managementKey?: string) => ({
       sdk.httpClient.post(apiPaths.authz.targetAll, { target }, { token: managementKey }),
       (data) => data.relations,
     ),
+  /**
+   * Return the list of all relations for the given target including derived relations from the schema tree.
+   *
+   * @param target The target to check relations for
+   * @returns array of relations that exist for the given targets
+   */
+  getModified: (since: Date): Promise<SdkResponse<AuthzModified>> =>
```

Comment:
perhaps we want to add it to REDAME as well

Reply:
Not sure what to write there - it can be confusing to users because it involves external stuff like caches